src/com/form.js

```diff
@@ -164,9 +164,12 @@
 			t = $(t);
 			var field = App.components.create('form-field', options.fieldsOptions);
 
+			var showFirstErrorOnly = t.filter('[data-only-show-first-error]').length === 1;
```

Comment:
exactly. is just to detect if the attribute is there or not without checking the value directly